.github/workflows/auto-merge-dependabot-workflow.yaml

```diff
@@ -15,6 +15,14 @@ jobs:
       is_bot1: ${{ github.actor == 'dependabot[bot]' }}
       is_bot2: ${{ github.actor == 'dependabot-preview[bot]' }}
     steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@17d0e2bd7d51742c71671bd19fa12bdc9d40a3d6 # v2.8.1
+        with:
+          disable-sudo: true
+          egress-policy: audit
+          disable-telemetry: false
```

Comment:
@tomwillis608 just thinking..  I was reading the [telemetry documentation](https://docs.stepsecurity.io/harden-runner/how-tos/disable-telemetry) and what the implications of that may be for certain projects or contractual vehicles.

Reply:
Fair.  It is easiest to deduce the necessary egress point through the telemetry. One could disable telemetry, set it to blocking mode and iteratively add the blocked sites without sending the telemetry to Step Security. You would miss out on other insights that happen with the data from the telemetry.

We should study how to use this in a more locked down environment.